src/main/kotlin/com/sourcegraph/cody/context/EmbeddingStatusView.kt

```diff
@@ -103,7 +103,7 @@ class EmbeddingStatusView(private val project: Project) : JPanel() {
   }
 
   fun setOpenedFileName(fileName: String, filePath: String?) {
-    openedFileContent.setText(fileName)
+    openedFileContent.text = fileName
     openedFileContent.setToolTipText(filePath)
```

Comment:
Why you only changed one setter and not the other? :)

Reply:
fixed